ui/admin/app/routes/scopes/scope/roles/role/manage-scopes/manage-custom-scopes.js

```diff
@@ -40,27 +40,96 @@ export default class ScopesScopeRolesRoleManageScopesManageCustomScopesRoute ext
       scope_id: [{ equals: scope_id }],
     };
 
-    const subScopes = await this.store.query('scope', {
+    const orgScopes = await this.store.query('scope', {
       scope_id,
       query: { search, filters },
       page,
       pageSize,
     });
-    const totalItems = subScopes.meta?.totalItems;
+    const totalItems = orgScopes.meta?.totalItems;
     const totalItemsCount = await this.getTotalItemsCount(
       scope_id,
       search,
       totalItems,
     );
 
+    const projectsSelected = await this.getProjectsSelected(
+      role.grantScopeProjectIDs,
+      orgScopes,
+    );
+
     return {
       role,
-      subScopes,
+      orgScopes,
+      projectsSelected,
       totalItems,
       totalItemsCount,
     };
   }
 
+  async getProjectsSelected(projectIDs, orgScopes) {
+    const options = { pushToStore: false, peekIndexedDB: true };
+    const projectsSelected = {};
+
+    const projectScopes = await Promise.all(
+      orgScopes.map(({ id: scope_id }) =>
+        this.store.query(
+          'scope',
+          {
+            scope_id,
+            query: {
+              filters: {
+                scope_id: [{ equals: scope_id }],
+              },
+            },
+            page: 1,
+            pageSize: 1,
+          },
+          options,
+        ),
+      ),
+    );
+
+    projectScopes.forEach((items) => {
+      if (items.length)
+        projectsSelected[items[0].scope.id] = {
+          selected: 0,
+          total: items.meta?.totalItems,
+        };
+    });
+
+    if (projectIDs.length) {
+      const id = [];
+      projectIDs.forEach((proj_id) => id.push({ equals: proj_id }));
```

Comment:
I think we normally camel case this instead of snake case

Reply:
oh yeah, good catch